dashboard/test/ui/test_status.js

```diff
@@ -171,7 +172,7 @@ function testFromS3Key(key) {
   // unlikely.
   var test = tests[browser][feature];
   while (!test && feature.indexOf('_') >= 0) {
-    feature = feature.replace('_', '/'); // Replaces first underscore only.
+    feature = feature.replace(/_/, '/'); // Replaces first underscore only.
```

Comment:
Huh. I could have sworn string needles only ever replaced the first instance; could this be an ES6 thing? Perhaps babel transpiling something it shouldn't?